tests/release/release-failures.go

```diff
@@ -34,6 +38,16 @@ var _ = framework.ReleaseSuiteDescribe("test-release-service-failures", func() {
 			// Create the managed namespace
 			_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
 			Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", managedNamespace, err)
+
+			// Wait until the "pipeline" SA is created and ready with secrets by the openshift-pipelines operator
+			klog.Infof("Wait until the %s SA is created in %s namespace \n", "pipeline", managedNamespace)
```

Comment:
Pipeline is not in a variable so doesn't make much sense to use variable substitution. You can hardcode the value directly or move it to a variable.
```suggestion
			klog.Infof("Wait until the 'pipeline' SA is created in %s namespace \n", managedNamespace)
```